stable/ibex-dashboard/templates/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClassName) -}}
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
Why not `storage.k8s.io/v1` here?

Reply:
`v1` rather than `v1beta1` should be used since both 1.7 and 1.8 support it. When 1.9 comes out (in just over a month) `v1beta1` will no longer be supported unless it's extended longer than the deprecation policy.

Reply:
@scottrigby @mattfarina I fixed it, and also added newline to eof.